cleanlab/count.py

```diff
@@ -862,6 +874,11 @@ def estimate_py_noise_matrices_and_cv_pred_proba(
     clf_kwargs : dict, optional
       Optional keyword arguments to pass into `clf`'s ``fit()`` method.
 
+    validation_func : callable, optional
+      Optional callable function that takes two arguments, `X_val`, `y_val`.
+      Specifies how to map the validation data split in the cross validation
+      step into the appropriate format to pass into `clf`'s ``fit()`` method.
+
```

Comment:
Since I suggested many additions to the docstring for this in CleanLearning.fit(), here I'd now just say:

```
validation_func : callable, optional
    Specifies how to map the validation data split in cross-validation as input for ``clf.fit()``.  For details, see the documentation of :py:meth:`CleanLearning.fit<cleanlab.classification.CleanLearning.fit>`

```

Reply:
done!